apiserver/server.go

```diff
@@ -22,6 +22,8 @@ type Config struct {
 	Logger lager.Logger
 	// EnablePanic will cause the server to crash on panic if set to true
 	EnablePanic bool
+	// Flag to indicate if only the health check should be exposed
+	StatusOnly bool
```

Comment:
I guess this name makes as much sense as any in context of everything else calling this `status`.

Reply:
Want me to rename status here and the function call to healthz? I'd be tempted to change the url from / to healthz as well.

Reply:
Okay can do. Let me make the update.

Reply:
Done